dev/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@types/react": "^16.14.2",
     "@types/react-dom": "^16.9.10",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.6",
```

Comment:
When updating `package.json` please also update the lock file, since otherwise they won't be in sync.